CHANGELOG.md

```diff
@@ -133,6 +139,7 @@ Huge thanks goes out to all of our contributors for this release:
 - Valentin Fedoskin
 - yfei1
 
+>>>>>>> origin/master
```

Comment:
I remember you took this down. It came back! Make sure you didn't lost whatever changes made at some point.

Reply:
Weird: it wasn't flagged as a conflict.